Makefile

```diff
@@ -19,4 +19,7 @@ build-docker:
 test:
 	go test -v ./...
 
+stress:
+	go test  -run  Example -count 10000
```

Comment:
Maybe convert to a benchmark?

Reply:
What I care about is trying to make the code "throw" some unwanted exceptions.

Reply:
Yeah, a  benchmark will do that :)